src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -3370,7 +3370,7 @@ private boolean sendWarnings(KoLAdventure adventure, String adventureName, Strin
     String limitmode = KoLCharacter.getLimitmode();
 
     // If we are playing Spelunky, a specialized set of warnings are relevant
-    if (limitmode.equals(Limitmode.SPELUNKY)) {
+    if ((limitmode != null) && limitmode.equals(Limitmode.SPELUNKY)) {
```

Comment:
I don't think you need these parentheses.

As an alternative, you could switch the order of comparison:

SPELUNKY.equals(limitmode)

Reply:
As a defensive measure I tend to use extra parens instead of trying to recall the evaluation hierarchy.  

limitmode is declared as a String but null also is expected and has meaning.  This formulation makes that slightly more obvious, at least until limitmode becomes an Enum.

I'll leave it as is since that gets this fix out faster.  Thank you.